src/nandini/Assignment_44/FreqOfWord.java

```diff
@@ -0,0 +1,24 @@
+package nandini.Assignment_44;
+
+import java.util.HashMap;
+
+public class FreqOfWord {
+	
+	public static void main(String args[]) {
+		HashMap<String,Integer> map = new HashMap<>();
+		String str = "hello gm hi gm hello pune gn";
+		String [] strArray = str.split(" ");
+		for(int index = 0; index < strArray.length;index++) {
+			String word = strArray[index];
+			if(map.containsKey(word)) {
+				int cnt = map.get(word)+1;
```

Comment:
2nd approach, if interviewer ask
you can use below method as well, try to understand below mentioned method at your end
map.put(key, map.getOrDefault(key, 0) + 1);